autoload/vsnip/snippet/node/variable.vim

```diff
@@ -47,8 +52,9 @@ endfunction
 function! s:Variable.to_string() abort
   return printf('%s(name=%s, unknown=%s, resolved=%s)',
   \   self.type,
+  \   self.name,
   \   self.unknown ? 'true' : 'false',
-  \   self.resolve()
+  \   self.resolve({})
   \ )
```

Comment:
Currently, I was rough working for it to anyway to work.

We should organize the variable node implementation.